resources/options_values.md

```diff
@@ -0,0 +1,263 @@
+
```

Comment:
Let's add the first line to give credit and add a link to the original manual.

Reply:
ok!

Reply:
Updated :+1: